### PR TITLE
Add sub tasks for Step2

### PR DESCRIPTION
**Pull request by bianjiangapple**
more detailed tasks

### CODE REVIEW

File: tasks [hunk 1]
```diff
@@ -11,5 +11,8 @@ Before anything can be started, the following items must be finished.
 
 
 # Step 2
+# sub task 1
+
+# sub task 2
```
Comment (godunm) [inline tasks:16]: there is one more task: task 3, maybe you can add it as well?

**Review by godunm — APPROVED** [review]
thanks for your quick work.

approved after comment being resolved.

File: tasks (hunk 1 above)
Comment (bianjiangapple) [reply tasks:16]: done.

**Comment by bianjiangapple** [comment]
btw. when this can be merged?

**Pull request merged**